src/components/uploader/css/Uploader.css

```diff
@@ -241,7 +241,7 @@ tr.fl-uploader-file-state-uploaded button:before {
     display: block;
     overflow: hidden;
     height: 2em;
-    width: 7em;
+    width: 9em;
```

Comment:
I've tried this in Safari, Chrome, Firefox (Mac OS and Win 8.1), and IE 11 and found similar results with the spacing being too wide on the right hand side. Can you try it out again on your end and determine if what you are getting is specific to Linux. Also centring the text of the button may make it look less odd. 
![screen shot 2016-04-01 at 10 09 23 am](https://cloud.githubusercontent.com/assets/556024/14209408/088f6892-f7f2-11e5-8f6a-f46f12b4168b.png)